api/rpc/account/account.go

```diff
@@ -23,10 +34,17 @@ func (s *Server) SignUp(ctx context.Context, in *SignUpRequest) (out *SessionRep
 	if err != nil {
 		return nil, err
 	}
-	_, err = passlib.Hash(in.Password)
+	hash, err := passlib.Hash(in.Password)
```

Comment:
So, we are setting the password in the signup command? I thought we would set the password via the email verification link / verify command?